package com.trbtree.service.modules.branch.repository;

import com.trbtree.service.modules.branch.entity.Message;
import org.springframework.data.jpa.repository.JpaRepository;

import java.util.UUID;

public interface MessageRepository extends JpaRepository<Message, UUID> {
}
